sql-cloudant/src/main/scala/org/apache/bahir/cloudant/CloudantReceiver.scala

```diff
@@ -16,23 +16,20 @@
  */
 package org.apache.bahir.cloudant
 
-// scalastyle:off
-import scalaj.http._
-
 import play.api.libs.json.Json
+import scalaj.http._
 
+import org.apache.spark.SparkConf
 import org.apache.spark.storage.StorageLevel
 import org.apache.spark.streaming.receiver.Receiver
-import org.apache.spark.SparkConf
 
 import org.apache.bahir.cloudant.common._
-// scalastyle:on
 
 class CloudantReceiver(sparkConf: SparkConf, cloudantParams: Map[String, String])
     extends Receiver[String](StorageLevel.MEMORY_AND_DISK) {
-  lazy val config: CloudantConfig = {
+  lazy val config: CloudantChangesConfig = {
```

Comment:
From our conversation over Slack, let's not change the name of the `CloudantReceiver` class.